templates/compass.html

```diff
@@ -6,23 +6,64 @@
+                              title="Close Modal">&times;</span>
+                        <div class="column" style="flex:8%">
+                            <img src="/gui/img/duk.png"/>
+                        </div>
+                        <div class="column" style="flex:92%">
+                            <p id="duk-text" style="color: white"></p>
+                                <div style="background-color: var(--primary-background);padding: 15px;border-radius: 25px">
+                                <h4>Layer Generation</h4>
+                                <ol style="text-align: left">
+                                    <li>Click 'Generate Layer'</li>
+                                    <li>Click '+' to open a new tab in the navigator</li>
+                                    <li>Select 'Open Existing Layer'</li>
+                                    <li>Select 'Upload from local' and upload the generated layer file</li>
+                                </ol>
+                            </div>
+                            <br>
+                            <div style="background-color: var(--primary-background);padding: 15px;border-radius: 25px">
+                                <h4>Adversary Creation</h4>
+                                <ol style="text-align: left">
+                                    <li>Select Techniques: Right Click and 'add to selection' </li>
+                                    <li>Once all techniques are selected, change the score (bar chart under 'technique
+                                        controls' to a positive number</li>
+                                    <li>Select 'download layer as json' under layer controls</li>
+                                    <li>Upload Adversary Layer file. The Adversary will now be available to use in
+                                        an operation. The Adversary will be take on the name of the layer tab. The
+                                        adversary will be split into phases by tactic</li>
+                                </ol>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+             </form>
+          </div>
          <div id="stockpile" class="row-canvas">
             <div class="section-profile">
                     <div class="row">
+                        <div class="topright duk-icon"><img onclick="openHelp()" src="/gui/img/duk.png"></div>
                         <div class="column section-border" style="flex:37%;text-align:left;padding:15px;">
                             <h1 style="font-size:70px;margin-top:-20px;">Compass</h1>
-                            <h2 style="margin-top:-55px">find your way</h2>
-                            <br><br>
+                            <h2 style="margin-top:-50px">find your way</h2>
+                        </div>
+                        <div class="column section-border" style="flex:37%;text-align:left;padding:15px;">
```

Comment:
under the sub-header (find your way) I would add a <p> that says something like "you can generate a layer file for any adversary, which you can overlay on the matrix below. Vice-versa, you can create an adversary from the matrix itself." 

something like this that will explain to a newbie what a layer is.

Reply:
Updated